### PR TITLE
Basic benchmark mode

### PR DESCRIPTION
**Pull request by erivas**
Basic attempt to implement a benchmark mode (#407)

### CODE REVIEW

File: lib/Echidna/Config.hs [hunk 1]
```diff
@@ -153,6 +153,7 @@ instance FromJSON EConfigWithUsage where
                                  <*> v ..:? "initialize"      ..!= Nothing
                                  <*> v ..:? "multi-abi"       ..!= False
                                  <*> v ..:? "checkAsserts"    ..!= False
+                                 <*> v ..:? "benchmarkMode"    ..!= False
```
Comment (ggrieco-tob) [inline lib/Echidna/Config.hs:156]: Align this with the others `..!= False`

**Comment by arcz** [comment]
@erivas I think the code fine, could you add a test for this functionality?

File: examples/solidity/basic/benchmark.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+benchmarkMode: true
+coverage: true
+
```
Comment (ggrieco-tob) [inline examples/solidity/basic/benchmark.yaml:3]: Extra line, please remove it.

**Comment by erivas** [comment]
@arcz I've added a pair of tests.

**Comment by arcz** [comment]
@erivas could you rebase master? there are some conflicts

**Pull request merged**